host/xtest/regression_1000.c

```diff
@@ -2504,3 +2505,115 @@ static void xtest_tee_test_1034(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1034, xtest_tee_test_1034,
 		 "Test loading a large TA");
+
+struct bti_test {
```

Comment:
The whole test is not applicable to ARM32 or any other architecture that may later be supported. Should this be guarded by `#ifdef ARM64`? Or even `#ifdef CFG_TA_BTI`?

Reply:
Thanks, I will guard it by ARM64